vendor/github.com/contiv/libOpenflow/util/stream.go

```diff
@@ -35,6 +37,8 @@ type MessageStream struct {
 	pool *BufferPool
 	// Message parser
 	parser Parser
+	// Channel to shut down the parser goroutine
+	parserShutdown chan bool
```

Comment:
It's sent later on in the diff of this file:

https://github.com/contiv/libOpenflow/commit/d9c950917b660d7a8258ffe5842f863e065f7d2f#diff-16ba25332f097a22ef695fe81b87c6e7R90